core/src/test/scala/org/broadinstitute/dsde/rawls/snapshot/SnapshotServiceSpec.scala

```diff
@@ -10,15 +10,7 @@ import org.broadinstitute.dsde.rawls.dataaccess.datarepo.DataRepoDAO
 import org.broadinstitute.dsde.rawls.dataaccess.slick.TestDriverComponent
 import org.broadinstitute.dsde.rawls.dataaccess.workspacemanager.WorkspaceManagerDAO
 import org.broadinstitute.dsde.rawls.mock.MockDataRepoDAO
-import org.broadinstitute.dsde.rawls.model.{
-  DataReferenceDescriptionField,
-  DataReferenceName,
-  NamedDataRepoSnapshot,
-  RawlsRequestContext,
-  SamResourceAction,
-  SamResourceTypeNames,
-  SamUserStatusResponse
-}
+import org.broadinstitute.dsde.rawls.model.{DataReferenceDescriptionField, DataReferenceName, NamedDataRepoSnapshot, RawlsRequestContext, SamResourceAction, SamResourceTypeNames, SamUserStatusResponse}
```

Comment:
I'm not sure why my formatter smooshed all these into one line -- I liked it better before, but lost the battle with the IDE...any suggestions of what I did wrong here are welcome.

Reply:
This was actually a problematic change in that it was causing `scalafmt` to fail after pushing up the commit.  I'm not sure why, but updating my "Class count to use import with '_'" made the IDE stop smooshing the imports.

![](https://files.slack.com/files-pri/T0166C33N6A-F0608QVTZUY/image.png)